src/txadmission.cpp

```diff
@@ -578,8 +578,8 @@ bool AcceptToMemoryPool(CTxMemPool &pool,
 
     bool res = false;
 
-    CORRAL(txProcessingCorral, CORRAL_TX_PAUSE);
-    CommitTxToMempool();
+    TxAdmissionPause SerializeAcceptToMemoryPool;
+    LOCK(cs_main);
 
```

Comment:
why lock cs_main here....seems to be old behaviour maybe accidental?

Reply:
removed